### ISSUE TITLE
训练coco出现的问题

### ISSUE THREAD

**Issue by hdasd** [MVIG-SJTU/AlphaPose#731]
您好。我想请问下在windows环境中训练coco数据集的时候，出现如下问题，但我找不到解决方法。在出现第一个epoch后，程序自动终止退出，显卡明显没有调用。
由于gitbash无法进入虚拟环境，所以训练是在终端输入命令：
>python scripts/train.py --cfg configs/coco/resnet/256x192_res50_lr1e-3_1x.yaml
其中
num_worker=0
batch_size= 4
gpu ：nvidia 1660s
输出结果如下：
******************************
Namespace(board=True, cfg='configs/coco/resnet/256x192_res50_lr1e-3_1x.yaml', debug=False, detector='yolo', device=device(type='cuda', index=0), dist_backend='nccl', dist_url='tcp://192.168.1.214:23345', exp_id='default', gpus=[0], launcher='none', map=True, nThrea
ds=0, rank=-1, snapshot=2, sync=False, work_dir='./exp/default-256x192_res50_lr1e-3_1x.yaml/', world_size=4)
******************************
{'DATASET': {'TRAIN': {'TYPE': 'Mscoco', 'ROOT': 'C:\\\\project\\\\data\\\\coco', 'IMG_PREFIX': 'train2017', 'ANN': 'annotations\\\\person_keypoints_train2017.json', 'AUG': {'FLIP': True, 'ROT_FACTOR': 40, 'SCALE_FACTOR': 0.3, 'NUM_JOINTS_HALF_BODY': 8, 'PROB_HALF_
BODY': -1}}, 'VAL': {'TYPE': 'Mscoco', 'ROOT': 'C:/project/data/coco', 'IMG_PREFIX': 'val2017', 'ANN': 'annotations/person_keypoints_val2017.json'}, 'TEST': {'TYPE': 'Mscoco_det', 'ROOT': 'C:/project/data/coco', 'IMG_PREFIX': 'val2017', 'DET_FILE': './exp/json/test
_det_yolo.json', 'ANN': 'annotations/person_keypoints_val2017.json'}}, 'DATA_PRESET': {'TYPE': 'simple', 'SIGMA': 2, 'NUM_JOINTS': 17, 'IMAGE_SIZE': [256, 192], 'HEATMAP_SIZE': [64, 48]}, 'MODEL': {'TYPE': 'FastPose', 'PRETRAINED': '', 'TRY_LOAD': '', 'NUM_DECONV_F
ILTERS': [256, 256, 256], 'NUM_LAYERS': 50}, 'LOSS': {'TYPE': 'MSELoss'}, 'DETECTOR': {'NAME': 'yolo', 'CONFIG': 'detector/yolo/cfg/yolov3-spp.cfg', 'WEIGHTS': 'detector/yolo/data/yolov3-spp.weights', 'NMS_THRES': 0.6, 'CONFIDENCE': 0.05}, 'TRAIN': {'WORLD_SIZE': 4
, 'BATCH_SIZE': 32, 'BEGIN_EPOCH': 0, 'END_EPOCH': 200, 'OPTIMIZER': 'adam', 'LR': 0.001, 'LR_FACTOR': 0.1, 'LR_STEP': [90, 120], 'DPG_MILESTONE': 140, 'DPG_STEP': [20, 50]}, 'FILE_NAME': '256x192_res50_lr1e-3_1x.yaml'}
******************************
Create new model
=> init weights
loading annotations into memory...
Done (t=9.30s)
creating index...
index created!
############# Starting Epoch 0 | LR: 0.001 #############
  0%|                                                                                                         | 0/149813 [00:00<?, ?it/s]    

                                                                                                                                                                                                       
万分感谢！

更新:
调试发现是Segmentation fault，请问应该如何解决该问题？配置如下
alphapose        0.3.0+ad03490  
beautifulsoup4   4.9.3
cached-property  1.5.2
certifi          2020.6.20
chardet          3.0.4
colorama         0.3.9
crowdposetools   2.0
cycler           0.10.0
Cython           0.29.21
cython-bbox      0.1.3
easydict         1.9
future           0.18.2
google           3.0.0
h5py             3.1.0
idna             2.10
json-tricks      3.15.3
jsonpatch        1.26
jsonpointer      2.0
kiwisolver       1.3.1
matplotlib       3.3.2
munkres          1.1.4
natsort          7.0.1
nibabel          3.2.0
numpy            1.17.0
opencv-python    3.1.0.5
packaging        20.4
Pillow           8.0.1
pip              20.2.4
protobuf         4.0.0rc2
py               1.4.34
pycocotools      2.0
pyparsing        3.0.0b1
pytest           3.2.1
pytest-runner    2.11.1
python-dateutil  2.8.1
PyYAML           5.3.1
pyzmq            19.0.2
requests         2.24.0
scipy            1.1.0
setuptools       50.3.2
six              1.15.0
soupsieve        2.0.1
tensorboardx     2.1
terminaltables   3.1.0
timm             0.1.20
torch            1.4.0+cu92
torchfile        0.1.0
torchsample      0.1.3        
torchvision      0.5.0+cu92
tornado          6.1
tqdm             4.51.0
urllib3          1.25.11
visdom           0.1.8.9
websocket-client 0.57.0
wheel            0.29.0
wincertstore     0.2

**Comment by Fang-Haoshu** [comment]
你好， windows虚拟环境经常容易有问题，建议直接使用linux

**Comment by shelly-xue** [comment]
@Fang-Haoshu 您好，我用的linux16.04，训练卡在epoch 0，进度条一直是0，请问也是因为没有调用显卡的原因吗？
